README.rst

```diff
@@ -65,6 +65,10 @@ DOI:10.1021/acs.jproteome.5b00860*
 Download
 ********
 
+Install via pip
+
+    | pip install ursgal
+
```

Comment:
[Dev Meeting] Although very nice to finally install Ursgal via pip, the resource will not be installed straight away and then the explanation below how to install the resources won't work. We need to find an alternative.